<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Alcodess-MacBook-Air.local" failures="1" tests="9" name="CronberryTestng.CronberryTestng.integration.ViewAudience" time="111.999" errors="0" timestamp="2021-09-09T14:23:58 IST" skipped="0">
  <testcase classname="CronberryTestng.CronberryTestng.integration.ViewAudience" name="ViewAudienceOpen" time="11.392"/>
  <system-out/>
  <testcase classname="CronberryTestng.CronberryTestng.integration.ViewAudience" name="ViewAudienceDatePicker" time="13.325"/>
  <system-out/>
  <testcase classname="CronberryTestng.CronberryTestng.integration.ViewAudience" name="ViewAudienceReprt" time="10.735"/>
  <system-out/>
  <testcase classname="CronberryTestng.CronberryTestng.integration.ViewAudience" name="TotalLead" time="14.407">
    <failure message="expected:&lt;Total Count: 33&gt; but was:&lt;Total Count: 35&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected:<Total Count: 33> but was:<Total Count: 35>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:115)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:125)
at CronberryTestng.CronberryTestng.integration.ViewAudience.TotalLead(ViewAudience.java:155)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TotalLead -->
  <system-out/>
  <testcase classname="CronberryTestng.CronberryTestng.integration.ViewAudience" name="TotalFollowUPLead" time="16.461"/>
  <system-out/>
  <testcase classname="CronberryTestng.CronberryTestng.integration.ViewAudience" name="TotalCallLaterLead" time="10.381"/>
  <system-out/>
  <testcase classname="CronberryTestng.CronberryTestng.integration.ViewAudience" name="TotalConvertedLead" time="14.378"/>
  <system-out/>
  <testcase classname="CronberryTestng.CronberryTestng.integration.ViewAudience" name="TotalInterestedLead" time="10.557"/>
  <system-out/>
  <testcase classname="CronberryTestng.CronberryTestng.integration.ViewAudience" name="TotalwrongContactLead" time="10.363"/>
  <system-out/>
</testsuite> <!-- CronberryTestng.CronberryTestng.integration.ViewAudience -->
